### PR TITLE
Added PIP22 to resolve client agent pattern in GUI for windows

### PR DESCRIPTION
**Pull request by nagarajmanjunath**
* Added PIP22 to resolve client agent pattern in GUI for windows

### CODE REVIEW

**Comment by kehiy** [comment]
@pactus-project/research @nagarajmanjunath 

Most of the time an improvement proposal (especially for blockchain and network protocols) aims to be a general proposal for all implementations and clients. as I can see this proposal is only useful for our current GUI implementation on pactus-go core.

What do you think?

File: PIPs/pip-22.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+pip: 22
+title: Agent Node Defination
+author: Nagaraj (@ragnarok87)
+status: Draft
+type: Standards
+category: core
```
Comment (Ja7ad) [inline PIPs/pip-22.md:7]: Better change category to `Interface`

File: PIPs/pip-22.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+created: 29-01-2024
+---
```
Comment (Ja7ad) [inline PIPs/pip-22.md:9]: If this proposal it is related to another proposal add header:

```
requires: 14
```

**Review by Ja7ad — APPROVED** [review]

**Review by Ja7ad — CHANGES_REQUESTED** [review]
Please changes before merge!!!

**Review by Ja7ad — APPROVED** [review]

**Pull request merged**